src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. This will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
```

Comment:
"Error encountered while loading the project. Some project features, such as ... have been disabled."

You can fill out the ... with something truthful.  like "errors for closed files" and whatnot.